### ISSUE TITLE
Specify rules in YML-File

### ISSUE THREAD

**Issue by rruisong** [OpenSimulationInterface/osi-validation#28]
#### Describe the bug
We can define the rules for one type of data (i.e. osi-message) only once in yml-file. For example: If we only want to check 'SensorView.InterfaceVersion', the 'SensorView.GroundTruth.InterfaceVersion' will also be automatically checked. Because we can only define the rules for the type of data 'InterfaceVersion'

#### Describe how to reproduce the bug
1. Get the file '20200820_example_versionTest.osi' and 'osi_version.yml' from zip-file attached.
2. Move the file '20200820_example_versionTest.osi' to folder 'data'
3. Move the file 'osi_version.yml' to folder 'requirements-osi-3'
4. Run 'osivalidator -r -requirements-osi-3 .\data\20200820_example_versionTest.osi'
5. See output

#### Describe the expected behavior
We supposed to check if the major version of 'SensorView' is greater than 3 and didn't mean to check the major version of 'SensorView.GroundTruth'. We would define the two different kinds of version separately.

#### Show some screenshots
![image](https://user-images.githubusercontent.com/70255526/91282910-3b9efe00-e78a-11ea-89e5-59c6d9730d61.png)

#### Describe the OS you are using
- OS: Windows 10
- Language: Python
- Version 3.7

#### Additional context
Please get the osi- and yml-file from this zip-file.
[ForOsiIssue.zip](https://github.com/OpenSimulationInterface/osi-validation/files/5128728/ForOsiIssue.zip)

**Comment by vkresch** [comment]
That's a good point! This should "normally" be solved by keeping the `osi_version.yml` as is and add rules to `osi_sensorview.yml` like this:
```
SensorView:
  version:
    version_major:
      - is_greater_than: 3
```
which does not generate an output at all hence there is a bug somewhere in the tree parser implementation which needs to be solved. I will take a more detailed look when I have more time. Thanks for the report!

**Comment by rruisong** [comment]
Yes, I tried this "normal" way to add the rules as well. As you mentioned, it doesn't generate the corresponding output. Thank you for the confirmation!
